measurements/regard/regard.py

```diff
@@ -0,0 +1,159 @@
+>>> group2 = ['boys make too much noise', 'men are often violent']
+>>> results = regard.compute(predictions = group1, references = group2, aggregation = "maximum")
+>>> print({k: round(v[1],3) for k,v in results.items()})
+{'max_predictions_regard': 0.95, 'max_references_regard': 0.976}
+
+Example 3 (returns the average toxicity score):
+
+>>> regard = evaluate.load("regard", module_type="measurement")
+>>> group1 = ['the girls were mean', 'women are too ambitious']
+>>> group2 = ['boys make too much noise', 'men are often violent']
+>>> results = regard.compute(predictions = group1, references = group2, aggregation = "average")
+>>> print({k: round(v, 2) for k, v in results['average_predictions_regard'].items()})
+{'neutral': 0.37, 'negative': 0.57, 'other': 0.05, 'positive': 0.01}
+>>> print({k: round(v, 2) for k, v in results['average_references_regard'].items()})
+{'negative': 0.94, 'other': 0.04, 'neutral': 0.03, 'positive': 0.0}
+"""
+
+
+def regard(preds, refs, regard_classifier):
+    pred_scores = defaultdict(list)
+    ref_scores = defaultdict(list)
+    pred_regard = regard_classifier(preds)
+    ref_regard = regard_classifier(refs)
+    for pred in pred_regard:
+        for pred_score in pred:
+            pred_scores[pred_score["label"]].append(pred_score["score"])
+    for ref in ref_regard:
+        for ref_score in ref:
+            ref_scores[ref_score["label"]].append(ref_score["score"])
+    return dict(pred_scores), dict(ref_scores)
+
+
+@evaluate.utils.file_utils.add_start_docstrings(_DESCRIPTION, _KWARGS_DESCRIPTION)
+class Regard(evaluate.Measurement):
+    def _info(self):
+        return evaluate.MeasurementInfo(
+            module_type="measurement",
+            description=_DESCRIPTION,
+            citation=_CITATION,
+            inputs_description=_KWARGS_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "predictions": datasets.Value("string", id="sequence"),
+                    "references": datasets.Value("string", id="sequence"),
+                }
+            ),
+            codebase_urls=[],
+            reference_urls=[],
+        )
+
+    def _download_and_prepare(self, dl_manager):
+        regard_tokenizer = AutoTokenizer.from_pretrained("sasha/regardv3")
+        regard_model = AutoModelForSequenceClassification.from_pretrained("sasha/regardv3")
+        self.regard_classifier = pipeline(
+            "text-classification", model=regard_model, top_k=4, tokenizer=regard_tokenizer, truncation=True
+        )
+
+    def _compute(
+        self,
+        predictions,
+        references,
+        aggregation=None,
+    ):
+        pred_regard, ref_regard = regard(predictions, references, self.regard_classifier)
+        pred_average, ref_average = {}, {}
+        for k, v in pred_regard.items():
+            pred_average[k] = mean(v)
+        for k, v in ref_regard.items():
+            ref_average[k] = mean(v)
```

Comment:
Suggestion to simplify with dict comprehensions:

```suggestion
        pred_mean = {k: mean(v) for  k, v in pred_regard.items()}
        ref_mean = {k: mean(v) for  k, v in ref_regard.items()}
```